episodes/introduction.md

```diff
@@ -1,28 +1,246 @@
+Maybe an experiment reveals an edge case you hadn't considered.
+Maybe the analysis shows a suspicious gap that you could investigate with only a small modification to your approach.
+You might get almost to the end and find that some early assumption you made was complete nonsense that invalidated all of your results.
+Such setbacks are normal -- though heartbreaking -- and often necessary.
+Sometimes it's a quick trip backwards, and sometimes it's longer, but it is very common to learn more from something that doesn't quite work than from something that works perfectly on the first try.
+
+Publication is a great way to share what you have learned with others, and learn from others in turn.
+However,
+
+- False starts and wrong turns are not often included in research papers, even if a conference or journal would accept negative results.
+- When page limits are tight, details about the data processing and code are usually cut before details about the model and results.
+- While preprint venues can offer a way to collect feedback in advance of full peer review, it is almost always more focused on assumptions and approach than on troubleshooting a stack trace.
+- While many publication venues require code and data to be published alongside each paper, not all do, which can make replicating another lab's results difficult or even impossible.
+
+Introducing collaboration earlier in the research process can help with all of these problems.
+If you start your project with code sharing in mind, there will be less cleanup to do at publication time, and it will be easier to ask for advice from other researchers when you run into trouble.
+Most people are more willing to look at a nicely-organized and documented GitHub repository than wade into a 5,000-line Python notebook. It is also significantly easier to extract a minimum example of a problem if your code is already organized in functions.
+If you select source data that is publicly available, it is more likely that others will have seen and solved the same problems that you will encounter.
+There may even be existing tools to clean and process the data you are interested in -- or if not, publishing your cleaning and processing code independently of your research results can pay off on multiple axes. Other researchers will be able to replicate your results more easily if they can rely on a common starting point, you will help increase activity and accelerate development in your research area by reducing toil and duplicated effort, and your tooling can be a source of citations you may not have considered before.
+
+:::::::: challenge
+
+#### Introspection: durability
+
+Think of one of the well-established ideas or techniques in your field of research, something many people have built upon since it was first published. What do you think made that idea so useful? What helped it take hold? What helped it spread?
+
+::::::::
```

Comment:
the summary & collab paragraphs turned out long but did add an opportunity for this challenge. If we treat it as a pure script and don't elaborate it should be okay for time but we should pay attention to that in the dry-runs.